addons/quickmount/functions/fnc_getInNearest.sqf

```diff
@@ -75,12 +81,18 @@ if (!isNull _target &&
                 TRACE_2("",_effectiveRole,_x);
                 if (_effectiveRole != _desiredRole) exitWith {};
 
-                if (_role == "Turret") then {
-                    ACE_player action ["GetIn" + _role, _target, _turretPath];
-                    TRACE_3("Geting In",_x,_role,_turretPath);
+                if (!(_turretPath isEqualTo [])) then {
+                     // Using GetInTurret seems to solve problems with incorrect GetInEH params when gunner/commander
+                    ACE_player action ["GetInTurret", _target, _turretPath];
+                    TRACE_3("Geting In Turret",_x,_role,_turretPath);
                 } else {
-                    ACE_player action ["GetIn" + _role, _target];
-                    TRACE_3("Geting In",_x,_role);
+                    if (_cargoIndex > -1) then {
+                        ACE_player action ["GetInCargo", _target, _cargoIndex];
```

Comment:
Have you double checked this cargo index for vehicles with both FFV cargo turret slots and classic cargo slots? Like that one jeep from Apex? The cargo indices as used by some, but not all commands are shifted by the number of FFV slots.

Reply:
By Jeep I mean that buggy one. With the co driver machine gun.

Reply:
seems to work fine

Reply:
OK. Good!